docs/docs/forms.mdx

```diff
@@ -127,6 +127,40 @@ forms:
       not_intent: excluded_intent
 ```
 
+If the `from_entity` mapping is unique for some slot then
+it can also be used extract this slot from a given entity
+even if this slot wasn't requested by the form.
```

Comment:
What about simplifying this to:

'In `from_entity` mapping, only when an extracted entity uniquely maps onto a requested slot, the slot will be filled. The extracted entity will be ignored if the mapping is not unique."

Reply:
I'm not sure, since in your phrasing, it is confusing to me as if it'll not be used for requested slot

Reply:
rephrased using your suggestion